### PR TITLE
Add initial approvers for Traditional Chinese

### PR DESCRIPTION
**Pull request by seokho-son**
### Describe your changes
Add initial approvers for Traditional Chinese based on the initiation request (#1987)
- [A] Ader Fu (@ydFu)
- [A] Phil Huang (@pichuang)

### Related issue number or link (ex: `resolves #issue-number`)
#1987

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by seokho-son** [comment]
Hi, Ader Fu (@ydFu) Phil Huang (@pichuang) 
Welcome to the localization approver role!

please check and confirm following policies :)
[Policies for localization approvers (to be confirmed by candidates) #723](https://github.com/cncf/glossary/discussions/723)

**Comment by seokho-son** [comment]
PTAL @jihoon-seo @iamNoah1 @CathPag 
This PR is ready. :)

File: .github/settings.yml
Comment (jihoon-seo) [inline .github/settings.yml:146]: `zh-cn` would be correct 😅
( https://glossary.cncf.io/zh-cn/ )

```suggestion
  # l10n zh-cn approvers
```

File: .github/settings.yml
Comment (jihoon-seo) [inline .github/settings.yml:362]: For your convenience, I point out all existences of `zh-ch`:

```suggestion
  # l10n branch for zh-cn contents only
```

File: .github/settings.yml
Comment (jihoon-seo) [inline .github/settings.yml:371]:
```suggestion
        # zh-cn approvers
```

**Review by jihoon-seo — COMMENTED** [review]
@seokho-son PTAL!

File: .github/settings.yml
Comment (seokho-son) [reply .github/settings.yml]: Oh! Thanks. Updated.

**Review by jihoon-seo — APPROVED** [review]
LGTM! Welcome guys!

Edit: I don't have the permission to merge this PR.. So I cannot merge this one 😅
When merging, the merge method 'squash-and-merge' would be great. 😊

**Review by CathPag — APPROVED** [review]
LGTM 🎉

**Pull request merged**
